svf-llvm/include/SVF-LLVM/LLVMModule.h

```diff
@@ -250,19 +249,21 @@ class LLVMModuleSet
 
     const Value* getLLVMValue(const SVFBaseNode* value) const
     {
-        SVFBaseNode2LLVMValueMap ::const_iterator it = SVFBaseNode2LLVMValue.find(value);
-        assert(it!=SVFBaseNode2LLVMValue.end() && "can't find corresponding llvm value!");
-        return it->second;
-    }
-
-    const Value* getLLVMValue(const SVFBasicBlock* value) const
-    {
-        SVFBB2LLVMBBMap::const_iterator it = SVFBB2LLVMBB.find(value);
-        assert(it!=SVFBB2LLVMBB.end() && "can't find corresponding llvm value!");
-        return it->second;
+        if (value->getNodeKind() == SVFBaseNode::BasicBlockKd)
```

Comment:
use isa<>

Reply:
thanks for reminding. I write the classof()